core/gather/driver/execution-context.js

```diff
@@ -16,6 +16,13 @@ class ExecutionContext {
 
     /** @type {number|undefined} */
     this._executionContextId = undefined;
+    /**
+     * Marks the order that execution context ids are used, for purposes of having a unique
+     * value (that doesn't expose the actual execution context id) to
+     * use for __lighthouseExecutionContextUniqueIdentifier.
+     * @type {number[]}
+     */
+    this._executionContextIdentifiersUsed = [];
```

Comment:
what about a counter incremented in `_getOrCreateIsolatedContextId` when one is created?